domain/application/state/application_endpoint.go

```diff
@@ -229,7 +252,7 @@ FROM space`, spaceName{})
 		fromInput.Remove(space.Name)
 	}
 	if fromInput.Size() > 0 {
-		return internalerrors.
+		return errors.
 			Errorf("space(s) %q not found", strings.Join(fromInput.Values(), ",")).
```

Comment:
This goes against the grain of how we usually break lines. We would tend to do it after the `(` literal.